flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CollectionFunctionsITCase.java

```diff
@@ -417,6 +418,173 @@ private Stream<TestSetSpec> arrayReverseTestCases() {
                                         DataTypes.ROW(DataTypes.BOOLEAN(), DataTypes.DATE()))));
     }
 
+    private Stream<TestSetSpec> arraySliceTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.ARRAY_REVERSE)
```

Comment:
Don't we test `ARRAY_SLICE`?

Reply:
Ok, let me  change behavior of the function and rewrite the test.